cmd/osm-controller/osm-controller.go

```diff
@@ -159,7 +157,7 @@ func main() {
 		events.GenericEventRecorder().FatalEvent(err, events.InitializationError, "Error creating MeshSpec")
 	}
 
-	certManager, certDebugger, err := getCertificateManager(kubeClient, kubeConfig)
+	certManager, certDebugger, err := getCertificateManager(kubeClient, kubeConfig, cfg.IsDebugServerEnabled())
```

Comment:
This is a start, although the goal of PR #1740 is to make it possible to dynamically turn on/off the debug server via a ConfigMap.

This change does not achieve this because the DebugServer is still initialized in `main()`. The existing code must be refactored so that the debug server can be dynamically turned on/off via the ConfigMap toggle.

Reply:
There are many places where DebugServer is used. If you monitor the changes of DebugServer, the workload is very large.
I think we should create a new PR to solve the problem of hot loading, and even load all the parameters in ConfigMap.

Reply:
My response regarding this: https://github.com/openservicemesh/osm/issues/1740#issuecomment-698447422